src/PackageCard.tsx

```diff
@@ -76,15 +87,26 @@ const PackageCard: React.FC<PackageCardProps> = ({
       className={`${styles.packageCard} ${styles.opacityTransition} ${cardVisibilityClass}`}
     >
       <div className={styles.header}>
-        <h3 className={styles.title}>{packageDesc.friendlyName}</h3>
-        <p className={styles.unit}>(Downloads/Week)</p>
+        <div className={styles.headerLeft} />
+        <div className={styles.headerText}>
+          <h3 className={styles.title}>{packageDesc.friendlyName}</h3>
+          <p className={styles.unit}>(Downloads/Week)</p>
+        </div>
+        <div className={styles.headerControls}>
+          <Form.Switch
+            label="%"
+            checked={showAsPercentage}
+            onChange={() => setShowAsPercentage(!showAsPercentage)}
```

Comment:
Should this be something like:
```suggestion
            onChange={() => setMeasurementType(showAsPercentage ? "percentage" : "totalDownloads")}
```
It would make the change in `MemDownloadChart` nicer since you'll just pass the value directly as prop.